src/main/java/org/folio/innreach/specification/InnReachTransactionSpecification.java

```diff
@@ -37,16 +37,15 @@
   static Specification<InnReachTransaction> fieldsLookup(InnReachTransactionFilterParameters parameters) {
     return (transaction, cq, cb) -> {
       var hold = transaction.join("hold");
-      var itemHold = cb.treat(hold, TransactionItemHold.class);
-      var localHold = cb.treat(hold, TransactionLocalHold.class);
+      var transactionHold = cb.treat(hold, TransactionHold.class);
       var patronHold = cb.treat(hold, TransactionPatronHold.class);
 
       var typeIs = isOfType(cb, transaction, parameters);
       var stateIs = isOfState(cb, transaction, parameters);
       var centralCodeIn = centralCodeIn(cb, transaction, parameters);
       var patronAgencyIn = patronAgencyIn(cb, hold, parameters);
       var itemAgencyIn = itemAgencyIn(cb, hold, parameters);
-      var patronTypeIn = patronTypeIn(cb, itemHold, localHold, parameters);
+      var patronTypeIn = patronTypeIn(cb, transactionHold, parameters);
```

Comment:
probably patron name should be supported for filtering as well

Reply:
Fixed